mobile/src/main/java/com/ataulm/wutson/discover/ShowsInGenreAdapter.java

```diff
@@ -0,0 +1,69 @@
+package com.ataulm.wutson.discover;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.ataulm.wutson.R;
+
+class ShowsInGenreAdapter extends RecyclerView.Adapter<ShowsInGenreAdapter.ShowSummaryViewHolder> {
+
+    private final OnShowClickListener listener;
+    private final LayoutInflater layoutInflater;
+    private ShowsInGenre showsInGenre;
+
+    ShowsInGenreAdapter(LayoutInflater layoutInflater, OnShowClickListener listener) {
+        this.listener = listener;
+        this.layoutInflater = layoutInflater;
+    }
+
+    void update(ShowsInGenre showsInGenre) {
+        this.showsInGenre = showsInGenre;
+        notifyDataSetChanged();
+    }
+
+    @Override
+    public ShowSummaryViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        ShowSummaryView view = (ShowSummaryView) layoutInflater.inflate(R.layout.view_discover_shows_in_genre_item, parent, false);
```

Comment:
note the layout file is called `view_discover_shows_in_genre_item` because that's what's relevant at this point. Like if you had a list item that was just a TextView, we wouldn't call the layout `R.layout.view_text_view`